source/overview/corba.rst

```diff
@@ -0,0 +1,22 @@
+10 things you should know about CORBA
+=====================================
+
+1.  **You don’t need to know CORBA to work with TANGO**
+
+2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest **B**\ roker **A**\ rchitecture and it is a standard defined by the `Object Management Group (OMG) <www.omg.org>`_
```

Comment:
You may always try to build documentation locally before commit: `sphinx-build source/ build`, then open `build/index.html` with your browser and navigate to your document  ;-)